sheets/src/main/java/com/novoda/github/reports/sheets/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.novoda.github.reports.sheets;
+
+import com.novoda.github.reports.sheets.network.SheetsServiceClient;
+
+import rx.schedulers.Schedulers;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
this class will be eventually be dropped, atm it's just a convenience for testing